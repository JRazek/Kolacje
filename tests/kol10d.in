10 7
3 7 4 2 4 1 6 5 6 7
4 9 9
2 10 0
5 8 10
2 6 7
9 1 0
2 4 1
3 7 8
7 2 6
5 10 2
20
9 8 4
10 5 3
6 10 3
3 3 4
1 3 6
8 2 4
8 6 1
2 2 6
8 7 7
6 2 4
1 9 7
1 5 7
8 4 4
5 4 3
2 6 1
9 6 4
2 9 3
3 10 5
6 5 2
9 1 5
