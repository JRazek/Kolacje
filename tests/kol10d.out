22
22
27
0
24
12
19
12
18
11
20
12
13
21
7
21
10
38
11
44
